UKHO.PeriodicOutputService/UKHO.PeriodicOutputService.Common/Configuration/BessStorageConfiguration.cs

```diff
@@ -0,0 +1,10 @@
+﻿namespace UKHO.PeriodicOutputService.Common.Configuration
+{
+    public class BessStorageConfiguration
```

Comment:
As other comment, should the class be capitalised as 'BESSStorageConfiguration'

Reply:
keep it consistent with existing code.